learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
 
 <p class="text-muted">Some quick links and tips for increasing awareness of accessibility in digital content</p>
 
+
+---
+
+## Accessibility in DfE
+
+DfE have our own [Digital Accessibility Hub](https://educationgovuk.sharepoint.com/sites/lvewp00043) filled with helpful information on making the digital world more accessible.
+
+There are also the [DfE design manual](https://design.education.gov.uk), which has been created by the digital teams at DfE and includes [DfE's digital accessibility guidance](https://design.education.gov.uk/accessibility).
+
+---
+
+## Colour accessibility tools
+
+Contrast:
+
+- [WebAim (comparing two colours)](https://webaim.org/resources/contrastchecker/)
+- [Colour contrast matrix (comparing multiple colours](https://jansensan.github.io/color-contrast-matrix/)
+
+Colour blindness:
+
+- [Adobe colour accessibility (gives a pass / fail)](https://color.adobe.com/create/color-accessibility)
+- [Coblis (simulates colour blindness for uploaded images)](https://www.color-blindness.com/coblis-color-blindness-simulator/)
+- [ColourBlindCheck (R package for simulating colour blindness for a colour palette)](https://github.com/Nowosad/colorblindcheck)
+
+Grey scale:
+
+- You can often check colours in grey scale by using a print preview and switching the colour mode
+- [ColToGrey function within the DescTools R package](https://andrisignorell.github.io/DescTools/reference/ColToGrey.html)
+
+For more advice about colour in charts and visualisations specifically, including more tools and resources for checking colours yourself, see the [Analysis function colours in charts guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/#section-9).
+
+---
+
+## Free tools for reviewing web pages
+
+- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
```

Comment:
Chrome has a little c here and big C below

Reply:
Made this change in my PR comments commit (great spot!)